fall18/p3/README.md

```diff
@@ -1,3 +1,111 @@
+This project will focus on conditional statments and loops, to start download these files
+For those using pycharm, it may be easier to make the new project before downloading the files so you can save the files to the new directory created
+
+* [main.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p3/main.py)
+* [project.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p3/project.py)
+* [test.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p3/test.py)
+* [area.csv](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p3/hurricane.csv)
+
+A reminder: only make changes to the main.py file as that is the only one that you will be turning in
+
+after getting all the files into the same directory try running 
+
+```
+python test.py
+```
+
+you should see 
+```
+Your Python version: Python 3.6.5 :: Anaconda, Inc.
+
+Running your program with this command: python main.py
+
+RESULTS:
+  Function test 1 (testGetNumRecords): Got exception unsupported operand type(s) for -: 'NoneType' and 'int' when running test 1
+  Function test 2 (testGetName): function testGetName not found
+  Function test 3 (testGetWindSpeed): function testGetWindSpeed not found
+  Function test 4 (searchHurricane): function searchHurricane not found
+  Function test 5 (searchHurricane): function searchHurricane not found
+  Function test 6 (searchHurricane): function searchHurricane not found
+  Function test 7 (searchHurricane): function searchHurricane not found
+  Function test 8 (searchHurricaneByLocation): function searchHurricaneByLocation not found
+  Function test 9 (searchHurricaneByLocation): function searchHurricaneByLocation not found
+  Function test 10 (countHurricane): function countHurricane not found
+  Function test 11 (countHurricane): function countHurricane not found
+  Function test 12 (maxWindSpeedHurricane): function maxWindSpeedHurricane not found
+  Function test 13 (minWindSpeedHurricane): function minWindSpeedHurricane not found
+  Main function print test (Problem 1): expected (529) but found (0)
+  Main function print test (Problem 2): expected (BABE) but found (0)
+  Main function print test (Problem 3): PASS
+  Main function print test (Problem 4): expected (False) but found (0)
+  Main function print test (Problem 5): expected (JUNE) but found (0)
+  Main function print test (Problem 6): expected (241) but found (0)
+  Main function print test (Problem 7): expected (288) but found (0)
+  Main function print test (Problem 8): expected (65) but found (0)
+Score: 4%
+```
+
+#Directions
+
+The Order in which you should implement the functions are as follows
+
+##Step 1.
+implement the first three functions in the main.py file.
+
+These should be:
+
+testGetNumRecords()
+testGetName(index)
+testGetWindSpeed(index)
+
+These three functions are simple shells which will just test your ability to use the functions provided in the project.py file
```

Comment:
It is not very clear what students need to do for these three functions. They will not understand what shells mean. It might be better if we explain and show how to do Function 1 (as shown below) and ask them to do the other two in a similar way.
```python
def testGetNumRecords():
    return project.getNumRecords()
```